dashboards/lodestar_vm_host.json

```diff
@@ -148,7 +148,7 @@
           },
           "editorMode": "code",
           "exemplar": false,
-          "expr": "sum(process_resident_memory_bytes{job=~\"beacon|validator\"})",
+          "expr": "sum(process_resident_memory_bytes{job=~\"$beacon_job|beacon|$validator_job|validator\"})",
```

Comment:
what the point of matching both jobs? Can just remove job matcher

Reply:
@matthewkeil I looked at this again and I am not sure we want to track beacon and validator metrics in a single panel.

At least @philknows opened an issue for this previously, not sure what the desired behavior is now though.
- https://github.com/ChainSafe/lodestar/issues/5118

Regarding the 3rd `process_resident_memory_bytes` metric, seems to be from https://github.com/lidofinance/validator-ejector, this means job label matcher is required.

```
process_resident_memory_bytes{app="validator-ejector", client_name="lodestar", group="lido-goerli", host_type="lg1k", instance="lido-goerli-hzax41-0", job="ejector", network="goerli", scrape_location="ejector"}
```

Reply:
there is another job that I wanted to exclude.  only wanted to take those two